update-bot/wlmbots/checker_bot.py

```diff
@@ -121,14 +124,18 @@ def generate_category_result_table(self, results):
                 TemplateChecker.ERROR_MISSING_TEMPLATE: "",
                 TemplateChecker.ERROR_MISSING_IDS: "",
                 TemplateChecker.ERROR_INVALID_IDS: "",
-                TemplateChecker.ERROR_DUPLICATE_IDS: ""
+                TemplateChecker.ERROR_DUPLICATE_IDS: "",
+                TemplateChecker.ERROR_TOO_MANY_TEMPLATES: ""
             }
             severity = min(result["errors"].keys())
             errors.update(result["errors"])
+            if severity == 2 and errors[TemplateChecker.ERROR_TOO_MANY_TEMPLATES] > TemplateChecker.TEMPLATE_LIMIT_HARD:
+                severity = 1
```

Comment:
This is not *that* terrible but I find it a little bit confusing that until this point `severity` could be mapped to the type of error and now there's this one strange case where it doesn't.